### ISSUE TITLE
Images on homepage that have missing or unhelpful alt text

### ISSUE THREAD

**Issue by mark-a-wilson** [bcgov/digital_marketplace#168]
From [GDX accessibility audit](https://bcgov.sharepoint.com/:b:/t/00109-digital.gov.bc.ca/EQrwcYPvVqNPrpsUS_cmXPMBtgckjCybC3Nm02RqwMbRLQ?e=Jqz9Da) conducted in July 2021.

#### Issues
Use the [WAVE Evaluation Tool Chrome extension](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh) to see the areas of concern:

- BC Gov logo (header logo) should have descriptive alt text as it functions as a link
- Images for CWU & SWU boxes have alt text of "Code With Us Image" and "Sprint With Us Image". These don't help the user understand. They'd best be reverted to decorative (i.e. `alt=""`)
- Two other illustrations below don't have any alt attribute; they should also have decorative (i.e. `alt=""`)
